skt/reporter.py

```diff
@@ -513,48 +513,50 @@ def load_state_cfg(self, statefile):
 
         Raises: Exception if required 'runner' section is missing
         """
-        self.cfg = {}
+        cfg = {}
         state_to_report = ConfigParser.ConfigParser()
         state_to_report.read(statefile)
 
         # FIXME This can be simplified or removed after configuration and
         # state split
         for (name, value) in state_to_report.items('state'):
-            if not self.cfg.get(name):
+            if not cfg.get(name):
                 if name.startswith('jobid_'):
                     if 'jobs' not in self.cfg:
```

Comment:
I'm totally kicking myself for not noticing it sooner... but since you are changing the loading from `self.cfg` to `cfg`, it should be changed here (and on a bunch of other lines) too. Otherwise the reporting breaks. I think we need a test for `load_state_cfg` as well :D

Reply:
Ah, the docstring is broken, too. I'll go fix those.